nss-day-cohort-51/nss-day-cohort-51.github.io
# Mw feat create cta component 54

#54 
# Description

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Testing Instructions

- Fetch branch and run npm start to open website in browser
- Look for the white section immediately beneath the student list (DevList.
- You should see a single sentence with an invitation about Demo Day and a Demo Day button titled "RSVP"  

# Checklist:
